autopush/websocket.py

```diff
@@ -773,9 +773,14 @@ def _check_message_table_rotation(self, previous):
         self.transport.pauseProducing()
         # Check for table rotation
         cur_month = previous.get("current_month")
+        # Previous month user or new user, flag for message rotation and
+        # set the message_month to the router month
         if cur_month != self.ps.message_month:
-            # Previous month user or new user, flag for message rotation and
-            # set the message_month to the router month
+            if cur_month not in self.ps.settings.message_tables.keys():
```

Comment:
The 'in' test for Python dict by default checks the keys, calling .keys() on a dict isn't needed.